components/dashboard/src/projects/NewProject.tsx

```diff
@@ -729,6 +732,37 @@ function NewTeam(props: { onSuccess: (team: Team) => void }) {
         if (!teamName) {
             return;
         }
+
+        const isExperimentalTeamsServiceEnabled = await getExperimentsClient().getValueAsync(
+            "publicApiExperimentalTeamsService",
+            false,
+            {
+                user,
+            },
+        );
```

Comment:
I think we can use the `FeatureFlagContext` to get the value of this feature flag here.

Reply:
Good shout. Updated.